manifests/g/GaëtanSencie/Comptabilitéannuelle/1.7.0.0/GaëtanSencie.Comptabilitéannuelle.installer.yaml

```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GaëtanSencie.Comptabilitéannuelle

```

Comment:
While special characters are allowed, it may make it more difficult for users to find and install the package. I recommend considering omitting accents from the identifier such as `GaetanSencie.Comptabiliteannuelle`